init.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+# usage 1 (use go-rapisnark):
+# env ES_NODE_STORAGE_MINER=<miner> ./init.sh
+# usage 2 (use snarkjs, will check Node.js version and snarkjs installation):
+# env ES_NODE_STORAGE_MINER=<miner> ./init.sh --miner.zk-prover-impl 1
+
+if [ -z "$ES_NODE_STORAGE_MINER" ]; then
```

Comment:
There are many same code shared between shell script, could we include the same code as a common file?